test/views/MatchNotFoundViewSpec.scala

```diff
@@ -31,47 +31,47 @@ class MatchNotFoundViewSpec extends UnitSpec with RasTestHelper {
 
 	"match not found page" should {
 		"contain correct title when match not found" in {
-			val result = views.html.match_not_found("Jim McGill", dob, nino)(fakeRequest, testMessages, mockAppConfig)
+			val result = matchNotFoundView("Jim McGill", dob, nino)(fakeRequest, testMessages, mockAppConfig)
 			val doc = Jsoup.parse(contentAsString(result))
 			doc.title shouldBe Messages("match.not.found.page.title")
 		}
 
 		"contain customer details and residency status when match not found" in {
-			val result = views.html.match_not_found("Jim McGill", dob, nino)(fakeRequest, testMessages, mockAppConfig)
+			val result = matchNotFoundView("Jim McGill", dob, nino)(fakeRequest, testMessages, mockAppConfig)
 			doc(result).getElementById("match-not-found").text shouldBe Messages("member.details.not.found", "Jim McGill")
-			doc(result).getElementById("subheader").text shouldBe Messages("match.not.found.subheader","Jim McGill")
-			doc(result).getElementById("change-name").text shouldBe Messages("change.name") + " " + Messages("change")
-			doc(result).getElementById("name").text shouldBe "Jim McGill"
-			doc(result).getElementById("change-nino").text shouldBe Messages("change.nino") + " " + Messages("change")
-			doc(result).getElementById("nino").text shouldBe nino
-			doc(result).getElementById("change-dob").text shouldBe Messages("change.dob") + " " + Messages("change")
-			doc(result).getElementById("dob").text shouldBe dob
+			doc(result).getElementsByClass("govuk-body-l").text shouldBe Messages("match.not.found.subheader","Jim McGill")
+			doc(result).getElementById("change-name-link").text shouldBe Messages("change") + " " + Messages("change.name")
```

Comment:
Think im just curious about why these two messages have now swapped around and also what the context is ... as i read it as "Change Change name", think i need to visualise it on the page haha.

Reply:
One of them is a hidden element I believe!

Reply:
oooh okay, thats cheeky haha. Yep "visuallyHiddenText" in the view for change.name, cool.